pkg/build/builder/source.go

```diff
@@ -272,7 +279,7 @@ func copyImageSource(dockerClient DockerClient, containerID, sourceDir, destDir
 	if err := tempFile.Close(); err != nil {
```

Comment:
Would it be easier to just `tempFile.Close()` before returning the 270/277 error?
```
if err != nil {
   tempFile.Close()
   return err
}
```